src/plugins/gr-he-ansiblesetup/core/misc.py

```diff
@@ -266,6 +272,27 @@ def _customization(self):
                     default=_('No')
                 ) == _('Yes').lower()
 
+        if self.environment[
+            ohostedcons.CoreEnv.ENABLE_KEYCLOAK
+        ] is None:
+            self.environment[
+                ohostedcons.CoreEnv.ENABLE_KEYCLOAK
+            ] = dialog.queryBoolean(
+                dialog=self.dialog,
+                name='ovehosted_enable_keycloak',
+                note=_(
+                    '\nKeycloak integration with the engine is currently in '
+                    'Tech Preview.\n'
```

Comment:
why would it be TP? we don't really intend to test it more than we do right now. 
We did want to make it the default, we can say "Keyclock integration is now the default AAA in oVirt. To use the legacy internal AAA you have to answer "No"...

Reply:
That's how I personally perceive it. But I don't mind about the texts. If/when we (including Artur) trust this code enough to make it the default, obviously it does not make sense to mark it TP.